zsales_class_git.clas.abap

```diff
@@ -69,8 +72,11 @@ CLASS ZSALES_CLASS_GIT IMPLEMENTATION.
              matnr,
              charg,
              werks,
-             matkl,   "New Change V1 - Branch 1
-             erdat    "New Change V2 - Branch 2
+             lgort,    "New Change V4 - Branch 5
+             vstel,    "New Change V4 - Branch 5
```

Comment:
Remove Shipping point - not required

Reply:
The above review comment has been fixed.. Please review again